Gem/Code/Source/Automation/ScriptReporter.cpp

```diff
@@ -619,6 +627,30 @@ namespace AtomSampleViewer
                                 ShowDiffButton("View Diff", screenshotResult.m_officialBaselineScreenshotFilePath, screenshotResult.m_screenshotFilePath);
                                 ImGui::PopID();
 
+                                if (screenshotResult.m_officialComparisonResult.m_resultCode == ImageComparisonResult::ResultCode::ThresholdExceeded && ImGui::Button("Export Png Diff"))
```

Comment:
Sometimes the user might want to export the diff even if the threshold did not fail. You could do something like the "Force Show 'Update' Buttons" button above to allow users to get the export buttons.